test/extensions/wasm/test_data/segv.cc

```diff
@@ -0,0 +1,12 @@
+// NOLINT(namespace-envoy)
```

Comment:
You need to add `segv.wasm` and `segv.wat` to this commit.

Reply:
done.

Reply:
How can I reproduce the failure you are seeing?  When I run bazel test wasm_test in envoy/test/extensions/wasm I get:

//test/extensions/wasm:wasm_test                                         PASSED in 2.4s 

Which is to say I am seeing the Exception caught.

Reply:
Yes, it works fine in mocked tests (since you're explicitly catching Exception using `EXPECT_THROW_WITH_MESSAGE`), but that has nothing to do with "normal" usage.

To reproduce that stacktrace, you can add `badptr` code to any WASM HTTP filter (e.g. `headers.cc`), recompile it, and start Envoy with it. Envoy should crash when executing `badptr` code.

Reply:
Ah, yes, from the unhandled Exception.  I was thinking that generally we would convert all errors of this type to Exceptions so that we can catch them and signal  the controller (istio) then unload/disable the filter but I haven't completed that code yet. I was expecting to do it in a later PR.  I can capture the stack pointer/trace when I throw the Exception as well.   That should generalize to this case as well as those covered by Runtime::catchRuntimeExceptions().  Do you have another design in mind.  Let's talk offline.